[docs]: Create a troubleshooting section (#219)
hyperledger/iroha-2-docs: src/guide/troubleshooting/overview.md

# Troubleshooting

> This section is intended to help you if you encounter issues while working

Reviewer: ```suggestion
This section is intended to help if you encounter issues while working
```

with Iroha. If something goes wrong, please
[check the keys](#check-the-keys) first. If that doesn't help, check

> troubleshooting instructions for each stage:

Reviewer: ```suggestion
the troubleshooting instructions for each stage:
```

- [Installation issues](./installation-issues.md)
- [Configuration issues](./configuration-issues.md)
- [Deployment issues](./deployment-issues.md)
- [Integration issues](./integration-issues.md)

If the issue you are experiencing is not described here, contact us via
[Telegram](https://t.me/hyperledgeriroha).

## Check the keys

> Most issues arise as a result of the keys not matching. This is why we

Reviewer: ```suggestion
Most issues arise as a result of unmatched keys. This is why we
```

Reviewer: This one is optional

recommend to follow this rule: **If something goes wrong, please check the
keys first.**

> Here's a quick explanation. It is not possible to differentiate the error

Reviewer: ```suggestion
Here's a quick explanation: It is not possible to differentiate the error
```

> messages that arise when the keys of peers are not matching the keys in the

Reviewer: ```suggestion
messages that arise when peers' keys do not match the keys in the
```

array of

> [trusted peers](./../configure/peer-configuration.md#trusted-peers) because

Reviewer: ```suggestion
[trusted peers](./../configure/peer-configuration.md#trusted-peers), because
```

> it would expose the public key of the peers. As such, if you have Helm

Reviewer: ```suggestion
it would expose the peers' public key. As such, if you have Helm
```

> charts, K8s deployments with keys defined via the environment variables,

Reviewer: ```suggestion
charts or K8s deployed with keys defined via the environment variables,
```
